compiler/quilt/tools/build.py

```
@@ -164,88 +164,109 @@ def _build_node(build_dir, package, name, node, fmt, target='pandas', checks_con
-        # environments = node.get(RESERVED['environments'])
-
-        checks = node.get(RESERVED['checks'])
-        if transform == ID:
-            #TODO move this to a separate function
-            if checks:
-                with open(path, 'r') as fd:
-                    data = fd.read()
-                    _run_checks(data, checks, checks_contents, name, rel_path, target, env=env)
-            if not dry_run:
-                print("Registering %s..." % path)
-                package.save_file(path, name, rel_path)
-        else:
-            # copy so we don't modify shared ancestor_args
-            handler_args = dict(ancestor_args.get(RESERVED['kwargs'], {}))
-            # local kwargs win the update
-            handler_args.update(node.get(RESERVED['kwargs'], {}))
-            # Check Cache
-            store = PackageStore()
-            path_hash = _path_hash(path, transform, handler_args)
-            source_hash = digest_file(path)
-
-            cachedobjs = []
-            if os.path.exists(store.cache_path(path_hash)):
-                with open(store.cache_path(path_hash), 'r') as entry:
-                    cache_entry = json.load(entry)
-                    if cache_entry['source_hash'] == source_hash:
-                        cachedobjs = cache_entry['obj_hashes']
-                        assert isinstance(cachedobjs, list)
-
-            # Check to see that cached objects actually exist in the store
-            # TODO: check for changes in checks else use cache
-            # below is a heavy-handed fix but it's OK for check builds to be slow  
-            if not checks and cachedobjs and all(os.path.exists(store.object_path(obj)) for obj in cachedobjs):
-                # Use existing objects instead of rebuilding
-                package.save_cached_df(cachedobjs, name, rel_path, transform, target, fmt)
+       
+        include_package = node.get(RESERVED['package'])
+        if include_package:
```

Maybe report an error if both `package` and `file` are present?